tests/unit/dynamodb/test_batch_write.py

```diff
@@ -0,0 +1,226 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Sort of inconsistent to call the file ``test_batch_write.py`` since the usual naming convention for these libraries is ``test_(module_name).py`` so ``test_table.py`` is what I would look for.